scripts/migrateHistoricBids.ts

```diff
@@ -127,6 +127,7 @@ function mapBidtoBidCancelledMessage(
       account: bid.account, //account instead of signer
       bidNonce: bid.bidNonce,
       version: bid.version ?? "1.0", //set version 1.0 by default
+      cancelDate: bid.cancelDate ?? 0 
```

Comment:
Why are you checking if `cancelDate` is undefined?

Reading the interface for `CancelledBid` i see:
```
export interface CancelledBid extends Bid {
  cancelDate: number;
}
```

Where `cancelDate` is a non-nullable type, but you are treating it as nullable.

Reply:
Sorry, just chalk that up to me still refreshing my knowledge of typescript. I guess I didnt realize number was already not nullable.